web/src/Backend.js

```diff
@@ -15,7 +15,7 @@ export function uploadTrace(action, websiteId, impressionId, data) {
 }
 
 export function getSessionId(websiteId) {
-  return fetch(`${Setting.ServerUrl}/api/get-session-id?websiteId=${websiteId}`, {
+  return fetch(`${Setting.ServerUrl}/api/get-session-id?websiteId=${websiteId}&userId=`, {

```

Comment:
Why this?

Reply:
The api `/api/get-session-id` is modified.

In [controller/session.go:](https://github.com/microsoft/mouselog/blob/e418725d1d91bd27f8d425653d041d621de6ed7c/controllers/session.go#L32)
```go
func (c *APIController) GetSessionId() {
	sessionId := c.StartSession().SessionID()

	trace.AddSession(sessionId, c.Input().Get("websiteId"), c.getUserAgent(), c.getClientIp(), c.Input().Get("userId"))

	c.Data["json"] = sessionId
	c.ServeJSON()
}
```